models/game.rb

```diff
@@ -0,0 +1,16 @@
+require ',/item'
```

Comment:
I would kindly suggest you pay more attention to details there is a typo that coma should be a dot
```
require './item'
```